ui/app/templates/components/wizard/replication-setup.hbs

```diff
@@ -11,15 +11,15 @@
       more interested in speed of access?
     </p>
 
-    <LearnLink @path="/vault/operations/ops-replication">
+    <DocLink @path="/vault/operations/ops-replication">
```

Comment:
[This page](https://developer.hashicorp.com/vault/operations/ops-replication) doesn't seem to exist

Reply:
Good catch! Thank you - the path should be `/vault/tutorials/enterprise/performance-replication`